Cargo.toml

```diff
@@ -25,6 +25,12 @@ rustc-dep-of-std = ["dep:compiler_builtins", "dep:core"]
 # i.e. avoid unconditionally enabling it in library crates.
 wasm_js = ["dep:wasm-bindgen", "dep:js-sys"]
 
+# Optional backend: custom-fallback
+# This flag allows a custom fallback backend.
+# This will be used as a last-resort instead of raising a compiler error when
+# no other backend is available.
+custom-fallback = []
```

Comment:
Change the documentation to say that this feature should *only* be enabled by a crate which provides a custom fallback.